.evergreen/orchestration/drivers_orchestration.py

```diff
@@ -228,6 +233,95 @@ def run_command(cmd: str, **kwargs):
     LOGGER.debug(f"Running command {cmd}... done.")
 
 
+def start_atlas(opts):
+    mo_home = Path(opts.mongo_orchestration_home)
+    image = f"docker.io/mongodb/mongodb-atlas-local:{opts.version}"
+    docker = shutil.which("docker") or shutil.which("podman")
+    stop(opts)
+    cmd = f"{docker} run --rm -d --name mongodb_atlas_local -p 27017:27017"
+    if opts.auth:
+        cmd += " -e MONGODB_INITDB_ROOT_USERNAME=bob"
+        cmd += " -e MONGODB_INITDB_ROOT_PASSWORD=pwd123"
+    if "podman" in docker:
+        cmd += " --health-cmd '/usr/local/bin/runner healthcheck'"
+    cmd += f" -P {image}"
+    container_id = subprocess.check_output(cmd, shell=True, encoding="utf-8").strip()
+    (mo_home / "container_id.txt").write_text(container_id)
+    # Wait for container to become healthy.
+    LOGGER.info("Waiting for container to be healthy...")
+    if "podman" in docker:
+        run_command(f"{docker} healthcheck run {container_id}")
+    cmd = f"{docker} inspect -f '{{{{.State.Health.Status}}}}' {container_id}"
+    tries = 0
+    while 1:
+        resp = subprocess.check_output(shlex.split(cmd), encoding="utf-8").strip()
+        if resp == "healthy":
+            break
+        if tries == 60:
```

Comment:
```suggestion
        if tries >= 60:
```
Improve safety of the termination condition.

Reply:
Bump: not addressed?